src/scene_shop.cpp

```diff
@@ -16,25 +16,46 @@
  */
 
 // Headers
-#include "game_temp.h"
 #include "game_system.h"
 #include "game_party.h"
 #include "input.h"
 #include "reader_util.h"
 #include "scene_shop.h"
 #include "output.h"
 
-Scene_Shop::Scene_Shop() {
+Scene_Shop::Scene_Shop(
+		std::vector<int> goods,
+		int shop_type,
+		bool allow_buy,
+		bool allow_sell,
+		Continuation on_finish)
+	: on_finish(std::move(on_finish))
+	  , goods(std::move(goods))
+	  , shop_type(shop_type)
+	  , allow_buy(allow_buy)
+	  , allow_sell(allow_sell)
```

Comment:
the indent looks weird. 2 Tabs for the args and then "," in front for the init list

Reply:
Fixed